ONSdigital/rm-case-service
# SUS005 - Change LMS case status

# Motivation and Context

Survey Enquiry Line (SEL) users need to be able to change the status of a social survey cases. This PR introduces these new case state transitions.

# What has changed



* State transitions
* Unit and Integration tests
* Small refactor for IAC service stub - moved to separate class so others can use it.


# How to test?



* Checkout rm-casesvc-api branch [sus005-change-lms-case-status](https://github.com/ONSdigital/rm-casesvc-api/pull/29)
* Edit the `pom.xml` from this branch for `casesvc-api` to point to `10.49.29-SNAPSHOT`
* Run `mvn clean install` to ensure all tests pass

# Links
